IBPSA/Fluid/Sources/PropertySource_T.mo

```diff
@@ -0,0 +1,77 @@
+within IBPSA.Fluid.Sources;
+model PropertySource_T
+  "Model for overriding fluid properties that flow through the component, using temperature input"
+  parameter Boolean use_T_in= false
+    "Set to true to get the leaving fluid temperature from the input connector"
+    annotation(Evaluate=true, Dialog(group="Inputs"));
+
+  extends IBPSA.Fluid.Sources.BaseClasses.PartialPropertySource;
```

Comment:
Move extends up before any parameter declaration

Reply:
This order is deliberate, otherwise the parameters are ordered in a non-logical sequence in the parameters window of dymola.